go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/ONSdigital/dp-component-test
 
-go 1.18
+go 1.19
 
 require (
-	github.com/ONSdigital/dp-mongodb-in-memory v1.3.1
+	github.com/ONSdigital/dp-mongodb-in-memory v1.5.0
```

Comment:
Some of the other lines in this require block need to be deleted as they contain unused dependencies (lines 8, 9, and 13): 
```
github.com/chromedp/cdproto v0.0.0-20211126220118-81fa0469ad77
github.com/chromedp/chromedp v0.7.6
github.com/smartystreets/goconvey v1.7.2
```